csm/core/services/alerts.py

```diff
@@ -216,6 +216,57 @@ async def retrieve_unresolved_by_node_id(self, node_id) -> Iterable[AlertModel]:
         query = Query().filter_by(alert_filter)
         return await self.db(AlertModel).get(query)
 
+    async def fetch_alert_for_support_bundle(self):
+        """
+        Fetches New and Active alerts except for IEM alerts.
+        Fetches alerts whose life cycle is completed(resovled + ack) for 7 days.
+        1. Fetching New alerts (resolved and ack both false)
+        """
+        combined_alert_list = []
+        new_alerts_list = await self.retrieve_by_range(
+            None, #time_range
```

Comment:
Does this query need time range ?

From comment above the code - Fetches alerts whose life cycle is completed(resovled + ack) for 7 days.

Reply:
For - 
1. New alerts -> No time range.
2. Active alerts -> No time range.
3. Alerts that are both (resolved + ack) -> Time range of 7 days

I have commented for the combined logic and then for the individual fetching like 1. Fetching New alerts (resolved and ack both false)